ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -378,8 +378,10 @@ let rec cmdtable_data : (string * cmd_spec) list =
   ; ( "pool-certificate-uninstall"
     , {
         reqd= ["name"]
-      ; optn= []
-      ; help= "Uninstall a pool-wide TLS CA certificate."
+      ; optn= ["force"]
```

Comment:
pool-certificate-uninstall is deprecated, so should I be updating it or leaving it as-is? My initial instinct was the latter, but they both use the same underlying function Cli_operations.pool_uninstall_ca_certificate and so the --force option functionality is included in pool-certificate-uninstall regardless.

Reply:
Please leave it as-is, same as the api call

Reply:
I've removed force from both this and `certificate_uninstall` in datamodel_pool.ml, but when testing pool-certificate-uninstall, force still works. This is expected because as I say, `certificate-uninstall` is using `Cli_operations.pool_uninstall_ca_certificate`, the same as `pool-uninstall-ca-certificate`. Is this what you intended? It adds functionality to the deprecated function but doesn't change the documentation.

(datamodel_pool.certificate_uninstall is completed unused AFAICT)